torchvision/prototype/transforms/functional/_deprecated.py

```diff
@@ -0,0 +1,71 @@
+import warnings
+from typing import Any
+
+import PIL.Image
+import torch
+
+from torchvision.prototype import features
+
+from ._meta import convert_color_space, convert_color_space_image_pil
+
+
+# TODO: this was copied from `torchvision.prototype.transforms._utils`. Given that this is not related to pytree / the
+#  Transform object, we should probably move it to `functional._utils`.
```

Comment:
I do not understand this TODO. Can we resolve it right now ?

Reply:
See https://github.com/pytorch/vision/pull/6474#pullrequestreview-1082060405.